### ISSUE TITLE
About training details or tricks

### ISSUE THREAD

**Issue by Aklice-new** [TencentARC/StereoCrafter#32]
Thanks for releasing such a greate work. I've reading the paper, but still confused with some problem when try to training. In the paper, only a few description about the traing details, such as  only finetuning the spatial layers in Unet, the same loss as diffusion and the learning rate. Can you share more details or tricks about training ? 

Looking forwad to your reply.
